### ISSUE TITLE
Code completion doesn't work with renamed, referred vars

### ISSUE THREAD

**Issue by Engelberg** [cursive-ide/cursive#1544]
``` clojure
(ns my-namespace
  (:require [clojure.set :as set :refer :all :rename {union uni}]))
```

This introduces a var `uni` into the current namespace that Cursive doesn't appear to know about.

**Comment by Engelberg** [comment]
If the thing you are renaming is a macro, this issue makes it impossible (as far as I can tell) to alter Cursive's indentation for the macro.

**Comment by marco-m** [comment]
@cursive-ide this impacts, for example, better-cond https://github.com/Engelberg/better-cond/tree/v2.0.1.

**Comment by cursive-ide** [comment]
I believe the original issue was fixed some time ago, in #394. However, renamed vars incorrectly appeared with the original names in completion lists, which is now fixed.
